jetty-cdi/src/main/java/org/eclipse/jetty/cdi/CdiSpiDecorator.java

```diff
@@ -61,7 +62,7 @@
     private final MethodHandle _dispose;
     private final MethodHandle _release;
 
-    public CdiDecorator(ServletContextHandler context) throws UnsupportedOperationException
+    public CdiSpiDecorator(ServletContextHandler context) throws UnsupportedOperationException
```

Comment:
Now that we removed the checkstyle rule on consecutive upper case letters, should this now be called `CDISPIDecorator` ?

Reply:
Nah.  I'm happy with CdiSpi rather than CDISPI.  Our code has both styles, so that's fine.